mmseg/models/segmentors/encoder_decoder.py

```diff
@@ -62,15 +62,18 @@ def _init_auxiliary_head(self, auxiliary_head):
 
     def extract_feat(self, img):
         """Extract features from images."""
+        output = []
         x = self.backbone(img)
+        output.append(x)
         if self.with_neck:
             x = self.neck(x)
-        return x
+            output.append(x)
+        return tuple(output)
```

Comment:
as neck can be an identity module, the feature from neck is equal to the feature from backnone.